src/App.js

```diff
@@ -1,10 +1,55 @@
+import { Route, Routes } from "react-router-dom";
+import Mockman from "mockman-js";
 import "./App.css";
-import logo from "./logo.png";
+import Navbar from "./components/Navbar/Navbar";
+import ProductCard from "./components/ProductCard/ProductCard";
+import { useEffect, useState } from "react";
+import axios from "axios";
+import CategoryCard from "./components/CategoryCard/CategoryCard";
+import HeroImg from "./components/HeroImg/HeroImg";
+import CartCard from "./components/CartCard/CartCard";
 
 function App() {
-  return (
-    <div className="App">
-
+  const [products, setProducts] = useState([])
+  // const getData =async () => {
+  //     const res = await axios.get('api/products')
+  //     console.log( res.data.products)
+  //     setProducts(res.data.products)
+  //   }
+  //   ()=>getData();
+    
+  // console.log(products)
+  
+  // useEffect(() => {
```

Comment:
you can remove this commented code if not  needed.

Reply:
Have left those for checking the code.